index.html

```diff
@@ -20,10 +20,25 @@
       shape-rendering: crispEdges;
     }
 
+    .overviewaxis path,
+    .overviewaxis line{
+      fill: none;
+      stroke: #bbb;
+      shape-rendering: crispEdges;
+    }
+
+    .overviewaxis .tick text{
+      color: #ddd;
+    }
+
+    .duplicate .tick text{
```

Comment:
Could you make this a little more specific? Maybe `.timeline .duplicate .tick`. We should try to prevent accidentally changing other classes